<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PagePanel</class>
 <widget class="QWidget" name="PagePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="pageMsg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 0);</string>
   </property>
  </widget>
  <widget class="HomePage" name="pageHome" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
  </widget>
  <widget class="QWidget" name="pageMine" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 0, 0);</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HomePage</class>
   <extends>QWidget</extends>
   <header>HomePage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
